orbyter-spark-dev/README.md

```diff
@@ -0,0 +1,74 @@
+# manifoldai/orbyter-spark-dev:0.1
+
+Dockerfile for [manifoldai/orbyter-spark-dev:0.1](https://hub.docker.com/r/manifoldai/orbyter-spark-dev)
+
+## Usage
+
+To run a bash shell
+
+`
+docker run -it manifoldai/orbyter-spark-dev:3.3 bash
+`
+
+## Image overview
+
+For a complete list all packages, run `pip freeze` in a container. Below is a summary of
+the useful packages for ML development.
+
+System:
+
+* Ubuntu 20.04 LTS
+* Python 3.8.5
+
+### Included Packages :
```

Comment:
Would it make sense to have this list just link to the requirements.txt file? Seems like it could accidentally get out of date.

Reply:
Good call! We should probably make that change across the board (not just for the spark image).